scripts/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tsconfig.json",
+  "include": [
+    "."
+  ]
+}
```

Comment:
Missing newline

Reply:
And now I realize I can just enable an editor setting to add newline at the end of file on save ^^